go/runtime/scheduling/simple/txpool/api/api.go

```diff
@@ -0,0 +1,55 @@
+// Package api defines the transaction pool interfaces.
+package api
+
+import (
+	"fmt"
+
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/hash"
+	p2pError "github.com/oasisprotocol/oasis-core/go/worker/common/p2p/error"
+)
+
+var (
+	ErrCallAlreadyExists = fmt.Errorf("call already exists in pool")
+	ErrFull              = fmt.Errorf("pool is full")
+	ErrCallTooLarge      = p2pError.Permanent(fmt.Errorf("call too large"))
+)
+
+// TxPoolConfig is a transaction pool configuration.
+type TxPoolConfig struct {
```

Comment:
```suggestion
type Config struct {
```
(as the package will usually be imported as `txpool`)